### ISSUE TITLE
Accessibility: Worldcat pluslet: the tabs in the Worldcat Search are not in the tab order

### ISSUE THREAD

**Issue by sandbergja** [subjectsplus/SubjectsPlus#1360]
Steps to reproduce:

1. Create a guide with a Worldcat pluslet
2. Go to the public view of the guide
3. Using the tab key, try to set the focus on one of the tabs on the Worldcat Search box.  Note that the focus goes immediately to the search box instead.

This is because the tabs are <a> tags without hrefs, which are not focusable.  I believe the best practice would be to turn these into aria-compliant tabs (see https://www.w3.org/TR/wai-aria-practices-1.1/#tabpanel), which respond to the arrow keys to change tabs.  This would also give it a consistent keyboard interaction with other tabbed interfaces in SubjectsPlus.

Failing that, we should at least make these navigable with the Tab key.  I think that changing <a> to <button> and styling appropriately would be the simplest way to do this.  We could also add a href attribute or a tabindex="0".

See the section "Items that cannot receive keyboard focus" here: https://webaim.org/techniques/keyboard/#navigation
